sys/README.md

```diff
@@ -0,0 +1,21 @@
+# openxr-sys
+
+## Cloning
+
+If you can, clone the repo using `git clone --recurse-submodules`. 
+
+If you already cloned the repo without, you must run (from anywhere in the repo):
+1. `git submodules init`.
+2. `git submodules update`.
+
+## Updating the OpenXR SDK
+
+1. Update the OpenSR-SDK git submodule to a new commit hash or tag:
+  a. `cd openxrs/sys/OpenXR-SDK`.
+  b. `git up <tag/xxx or hash>`.
+  c. `cd ../..` to get back to the `openxrs` repo root and check that the 
+  submodule is updated (should be a single line change to what looks like a file
+  at `openxrs/sys/OpenXR-SDK`).
+2. `cd generator; cargo run --bin generator` to regenerate the `sys` crate.
+3. `cd ..` to go back to the `openxrs` repo root`.
+3. `cargo build && cargo test && cargo fmt` to find any issues that need fixing.
```

Comment:
`cargo run --bin generator` works from the root, and `cargo fmt` should be ran *before* `cargo build`.

Reply:
I couldn't get it to run from the `openxrs` repo root due to lines 34-35 of `generatror/src/main.rs`:
```
let mut sys_out = File::create("../sys/src/generated.rs").unwrap();
let mut hl_out = File::create("../openxr/src/generated.rs").unwrap();
 ```

These seem to assume PWD directory is always one of the children of `openxrs` repo root. 

It looks like line 27 of `generatror/src/main.rs` might have the solution by starting the path at `env!(CARGO_MANIFEST_DIR)` and then moving up based on that fixed path.

Reply:
Ah, good catch. No need to fix that in this PR. `cargo fmt` should still be ran before building though.